modules/prebidServerBidAdapter/index.js

```diff
@@ -615,14 +615,14 @@ const OPEN_RTB_PROTOCOL = {
 
       const imp = { id: adUnit.code, ext, secure: s2sConfig.secure };
 
-      const ortb2 = {...utils.deepAccess(adUnit, 'ortb2Imp.ext.data')};
+      const ortb2 = {...utils.deepAccess(adUnit, 'ortb2Imp.ext.prebid')};
```

Comment:
We can't get rid of ext.data. Need to look at the whole ext tree:

```
const ortb2 = {...utils.deepAccess(adUnit, 'ortb2Imp.ext')};
```

I imagine this might have consquences on the code below.

Reply:
I managed to extend the `ext` object with very little code changes and keeping the old logic mostly  the same